setup.sh

```diff
@@ -85,80 +66,31 @@ PERLROOT=$INST_PATH/lib/perl5
 
 # allows user to knowingly specify other PERL5LIB areas.
 if [ -z ${CGP_PERLLIBS+x} ]; then
-  export PERL5LIB="$PERLROOT"
+  PERL5LIB="$PERLROOT"
 else
-  export PERL5LIB="$PERLROOT:$CGP_PERLLIBS"
+  PERL5LIB="$PERLROOT:$CGP_PERLLIBS"
 fi
 
+export PERL5LIB=$PERL5LIB
+
 #add bin path for install tests
 export PATH=$INST_PATH/bin:$PATH
 
 #create a location to build dependencies
 SETUP_DIR=$INIT_DIR/install_tmp
 mkdir -p $SETUP_DIR
 
-cd $SETUP_DIR
-
-## grab cpanm and stick in workspace, then do a self upgrade into bin:
+#install cpanm
 get_file $SETUP_DIR/cpanm https://cpanmin.us/
 perl $SETUP_DIR/cpanm -l $INST_PATH App::cpanminus
-CPANM=`which cpanm`
-echo $CPANM
-
-PCAP=`perl -le 'eval "require $ARGV[0]" and print $ARGV[0]->VERSION' PCAP`
-if [[ "x$PCAP" == "x" ]] ; then
-  echo "PREREQUISITE: Please install PCAP-core before proceeding:"
-  echo "  https://github.com/cancerit/PCAP-core/releases"
-  exit 1;
-fi
-
-
-CGPVCF=`perl -le 'eval "require $ARGV[0]" and print $ARGV[0]->VERSION' Sanger::CGP::Vcf`
-if [[ "x$CGPVCF" == "x" ]] ; then
-  echo "PREREQUISITE: Please install cgpVcf before proceeding:"
-  echo "  https://github.com/cancerit/cgpVcf/releases"
-  exit 1;
-fi
-
-echo -n "Compiling pindel binaries ..."
-cd $INIT_DIR
-g++ -O3 -o $SETUP_DIR/pindel c++/pindel.cpp &&
-g++ -O3 -o $SETUP_DIR/filter_pindel_reads c++/filter_pindel_reads.cpp &&
-cp $SETUP_DIR/pindel $INST_PATH/bin/. &&
-cp $SETUP_DIR/filter_pindel_reads $INST_PATH/bin/. &&
-# convenience for testing
-mkdir -p $INIT_DIR/bin &&
-cp $SETUP_DIR/pindel $INIT_DIR/bin/. &&
-cp $SETUP_DIR/filter_pindel_reads $INIT_DIR/bin/.
-done_message "" "Failed during compilation of pindel."
-
-cd $INIT_DIR/perl
-
-echo -n "Installing Perl prerequisites ..."
-if ! ( perl -MExtUtils::MakeMaker -e 1 >/dev/null 2>&1); then
-    echo
-    echo "WARNING: Your Perl installation does not seem to include a complete set of core modules.  Attempting to cope with this, but if installation fails please make sure that at least ExtUtils::MakeMaker is installed.  For most users, the best way to do this is to use your system's package manager: apt, yum, fink, homebrew, or similar."
-fi
-
-$CPANM -v --mirror http://cpan.metacpan.org -notest -l $INST_PATH/ --installdeps . < /dev/null
-done_message "" "Failed during installation of core dependencies."
-
-echo -n "Installing cgpPindel ..."
-perl Makefile.PL INSTALL_BASE=$INST_PATH &&
-make &&
-make test &&
-make install
-done_message "" "cgpPindel install failed."
 
-# cleanup all junk
-rm -rf $SETUP_DIR
+bash build/opt-build.sh $INST_PATH
+bash build/opt-build-local.sh $INST_PATH
 
 echo
 echo
 echo "Please add the following to beginning of path:"
 echo "  $INST_PATH/bin"
-echo "Please add the following to beginning of PERL5LIB:"
```

Comment:
Users of the setup.sh route will still need to add to the perl5lib

Reply:
Done